src/test/rustdoc/sized_trait.rs

```diff
@@ -0,0 +1,15 @@
+#![crate_name = "foo"]
+
+// @has foo/struct.Bar.html
+// @has - '//h3[@id="impl-Sized"]' 'impl Sized for Bar'
```

Comment:
This seems not super helpful. Almost all types implement `Sized`. I think it would be more helpful to only show `impl !Sized`, which is outside the ordinary.

Reply:
That's a good point. Just to confirm: what do you think @rust-lang/rustdoc ?

Reply:
Agreed

Reply:
Yeah, and if we change our mind we can always add `impl Sized for Bar` back later.

Reply:
Fine by me! Let's go for it then.